testing/nightly/custom.sh

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env bash
+
```

Comment:
Could you add a description at the top of the file explaining what it's doing (overview)?

Reply:
fixed:
```
# This script is executed from `ci-artifacts` PR, when requesting
# `\test custom`.
#
# It looks at the message of the last commit of the PR (2nd parent of
# HEAD merge commit), and scans it for a test-path directive:
# > test-path: testing/.../test.sh [flags]
# and executes the test command.
#
# Calling this script on a commit without a `test-path` fails the test
# (exit 1).
```